front/src/applications/stdcm/views/StdcmRequestModal.tsx

```diff
@@ -48,71 +55,77 @@ export default function StdcmRequestModal(props) {
   // https://developer.mozilla.org/en-US/docs/Web/API/AbortController
   const controller = new AbortController();
 
-  const timetableId = osrdconf.timetableID;
+  const timetableId = osrdconf.timetableID?.toString();
 
   // Returns a promise that will be a fetch or an axios (through react-query)
   const stdcmRequest = async () => {
-    const params = formatStdcmConf(dispatch, setFailure, t, osrdconf);
-    return postStdcm(params).unwrap();
+    const params = formatStdcmConf(dispatch, setFailure, t, osrdconf as OsrdStdcmConfState);
+    return postStdcm(params as PostStdcmApiArg).unwrap();
```

Comment:
Not sure how it works in the app and if it can happen but should you add a try/catch in case params is undefined ?

Reply:
why not, but is it really necessary?

Reply:
I'm not sure maybe better to check it with a maintainer

Reply:
actually, if `undefined` you should not call `postStdcm` at all, it means one of the failures in `formatStdcmConf` happened. the error is handled in the reducer.
kinda missed that detail

Reply:
postStdcm accepts undefined too so you can pass it directly

```suggestion
    return postStdcm(params).unwrap();
```